### PR TITLE
{bio,lang}[system/system] Miniconda3 v4.12.0, QIIME2 v2022.8

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/23fea94c8e00a6645af7a72ffb372429 for a full test report.

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'QIIME2'
+version = '2022.8'
+
+homepage = 'http://qiime2.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
+ from raw DNA sequencing data."""
+
+source_urls = ['https://data.qiime2.org/distro/core/']
+sources = ['qiime2-%(version)s-py38-linux-conda.yml']
+checksums = ['454bb219f928c7f4ec0097888db27ab439ba2ef7e4294f38b7033327cb3dafb8']
```
Comment (branfosj) [inline easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb:16]: I'm getting `e43bffad08dfd8ab9054f1530e501258525a7b9a3f3079ac04fad701d797c90b` as the checksum for this. Does this mean that they've done an inplace change to that file?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb:16]: yes, me too now. looks like they re-uploaded the file.

Comment (maxim-masterov) [reply easybuild/easyconfigs/q/QIIME2/QIIME2-2022.8.eb]: updated checksum

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/91a92e779de3c6dc4997ba1e46c1190b for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**